esrfconcert/devices/motors/micos.py

```diff
@@ -83,42 +84,42 @@ def __init__(self, controller, index, host, port):
         super(LinearMotor, self).__init__()
         _Base.__init__(self, controller, index, host, port)
 
-    def _get_position(self):
-        return self._get_position_in_steps() * q.mm
+    async def _get_position(self):
+        return await self._get_position_in_steps() * q.mm
```

Comment:
This I was not sure about @tfarago 
I saw lines that looked rather like
return (await self._device['position']).value * self['position'].unit
anyway I figured I don't need to access the .value property here, so I don't need (...)

Reply:
Yes, this will work.